test/default.sh

```diff
@@ -1,4 +1,7 @@
-. $(dirname $0)/_functions.sh
+#!/bin/bash
```

Comment:
I think the shebang can be removed since these scripts are not executable.

Reply:
Replaced that with the shellcheck comment.